src/Components/SavedMethods.res

```diff
@@ -71,17 +72,18 @@ let make = (
           <Block bottomElement padding="px-4 py-1" className="max-h-[309px] overflow-auto" />
         </RenderIf>
       }}
-      <RenderIf condition={list.payment_methods->Js.Array.length !== 0}>
-        <DynamicFields
-          paymentType
-          list
-          paymentMethod="card"
-          paymentMethodType="debit"
-          setRequiredFieldsBody
-          isSavedCardFlow=true
-          savedCards=savedMethods
-        />
+      <RenderIf condition={billingAddress.isUseBillingAddress}>
```

Comment:
shouldnt this be added at the top level which will eliminate the redundant code in the card element too